sdk/ai/Azure.AI.Projects/src/Custom/Agent/AgentsClient.Streaming.cs

```diff
@@ -163,6 +165,96 @@ public virtual CollectionResult<StreamingUpdate> CreateRunStreaming(string threa
         Response sendRequest() => CreateRunStreaming(threadId, createRunRequest.ToRequestContent(), context);
         return new StreamingUpdateCollection(sendRequest, cancellationToken);
     }
+    /// <summary> Submits outputs from tools as requested by tool calls in a stream. Stream updates that need submitted tool outputs will have a status of 'RunStatus.RequiresAction'. </summary>
+    /// <param name="threadId"> Identifier of the thread. </param>
+    /// <param name="runId"> Identifier of the run. </param>
+    /// <param name="toolOutputs"> A list of tools for which the outputs are being submitted. </param>
+    /// <param name="cancellationToken"> The cancellation token to use. </param>
+    /// <exception cref="ArgumentNullException"> <paramref name="threadId"/>, <paramref name="runId"/> or <paramref name="toolOutputs"/> is null. </exception>
+#pragma warning disable AZC0015 // Unexpected client method return type.
+    public virtual CollectionResult<StreamingUpdate> SubmitToolOutputsToStream(string threadId, string runId, IEnumerable<ToolOutput> toolOutputs, CancellationToken cancellationToken = default)
+#pragma warning restore AZC0015 // Unexpected client method return type.
+    {
+        Argument.AssertNotNullOrEmpty(threadId, nameof(threadId));
+        Argument.AssertNotNullOrEmpty(runId, nameof(runId));
+        Argument.AssertNotNull(toolOutputs, nameof(toolOutputs));
+
+        SubmitToolOutputsToRunRequest submitToolOutputsToRunRequest = new(toolOutputs.ToList(), true, null);
+        RequestContext context = FromCancellationToken(cancellationToken);
+        using DiagnosticScope scope = ClientDiagnostics.CreateScope("AgentsClient.SubmitToolOutputsToStream");
+        scope.Start();
+        try
+        {
+            HttpMessage message = CreateSubmitToolOutputsToRunRequest(threadId, runId, submitToolOutputsToRunRequest.ToRequestContent(), context);
+            message.BufferResponse = false;
+            Response sendRequest() => _pipeline.ProcessMessage(message, context, CancellationToken.None);
+            return new StreamingUpdateCollection(sendRequest, cancellationToken);
+        }
```

Comment:
Can we create separate methods like [these](https://github.com/Azure/azure-sdk-for-net/blob/356fa48d83e33239c34ef72acff734b46fd5f681/sdk/ai/Azure.AI.Projects/src/Custom/Agent/AgentsClient.Streaming.cs#L259)?  

The reason is that if you have the same method signature in the custom type, the same method will not be generated, allowing you to update things like `BufferResponse = false` in your custom methods. For example, if you have a method signature similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/356fa48d83e33239c34ef72acff734b46fd5f681/sdk/ai/Azure.AI.Projects/src/Generated/AgentsClient.cs#L2298) in your custom code and if you regenerate the code, then this method will not be generated.

Reply:
Fixed.